src/executor/host.rs

```diff
@@ -1113,8 +1113,64 @@ impl ReadyToRun {
                 })
             }
             HostFunction::ext_crypto_ecdsa_generate_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_crypto_ecdsa_sign_version_1 => host_fn_not_implemented!(),
+            HostFunction::ext_crypto_ecdsa_sign_version_1 => {
+                let data = [0; 32];
+                data.copy_from_slice(
```

Comment:
is there a way to avoid copying here? or it will be unsafe? 🤔

Reply:
Yeah, you can call `as_bytes` on what `blake2b` returns: https://docs.rs/blake2-rfc/latest/blake2_rfc/blake2b/struct.Blake2bResult.html

Reply:
`as_bytes` returns `&[u8]`, I need `[u8; 32]`

Reply:
Ah I see. You can do `<[u8; 32]>::try_from(....as_bytes()).unwrap()`